src/api/MessageAccessories.ts

```diff
@@ -0,0 +1,41 @@
+export type AccessoryCallback = (props: Record<string, any>) => JSX.Element;
+export type Accessory = {
+    callback: AccessoryCallback;
+    position: number | null | undefined;
+};
+
+export const accessories = new Map<String, Accessory>();
+
+export function addAccessory(
+    identifier: string,
+    callback: AccessoryCallback,
+    position: number | null | undefined
```

Comment:
```suggestion
    position?: number
```
this way its optional